.sync.yml

```diff
@@ -1,30 +1,10 @@
 ---
 .travis.yml:
-  extras:
-  - rvm: 2.4.1
-    env: PUPPET_INSTALL_TYPE=agent CHECK=acceptance BEAKER_debug=true BEAKER_set=docker/centos-7
-    services: docker
-    sudo: required
-    bundler_args: --without development
-    dist: trusty
-  - rvm: 2.4.1
-    env: PUPPET_INSTALL_TYPE=agent CHECK=acceptance BEAKER_debug=true BEAKER_set=docker/debian-8
-    services: docker
-    sudo: required
-    bundler_args: --without development
-    dist: trusty
-  - rvm: 2.4.1
-    env: PUPPET_INSTALL_TYPE=agent CHECK=acceptance BEAKER_debug=true BEAKER_set=docker/ubuntu-14.04
-    services: docker
-    sudo: required
-    bundler_args: --without development
-    dist: trusty
-  - rvm: 2.4.1
-    env: PUPPET_INSTALL_TYPE=agent CHECK=acceptance BEAKER_debug=true BEAKER_set=docker/ubuntu-16.04
-    services: docker
-    sudo: required
-    bundler_args: --without development
-    dist: trusty
+  docker_sets:
```

Comment:
I am not sure if this syntax is correct

Reply:
A quick modulesync locally should be able to figure this out one second.

Reply:
Fixed it @bastelfreak